### ISSUE TITLE
Custom error message on required Entry field shows default

### ISSUE THREAD

**Issue by JeanLucEsser** [verbb/formie#981]
### Describe the bug

When using an entry field, setting a custom error message does not work as the default message "Please fill out this field." is shown no matter what.

Edit: Issue exists only on the front. If entering a submission via the CP, custom error message is shown.

Bonus question: where in the form settings or global settings can we change the default value of this error message?

### Steps to reproduce

1. Create a form with a required Entry field
2. Set a custom error message - The default error message is shown

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.1.3

### Plugin version

2.0.0

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0",
  "...": "..."
}
```

Then run `composer update`. 

As for the global setting for this, there isn't as such, but you can use [static translations](https://craftcms.com/docs/3.x/sites.html#static-message-translations) created in a `formie.php` file to modify this.

**Comment by engram-design** [comment]
Fixed in [2.0.1](https://github.com/verbb/formie/releases/tag/2.0.1)
